### ISSUE TITLE
Missing Dockerfile, jenkinsfile, Helm, and Git repo when creating a spring boot application

### ISSUE THREAD

**Issue by Riduidel** [jenkins-x/jx#1554]
### Summary 
When I try to create a Spring boot application, it doesn't create the `Dockerfile`, `Jenkinsfile`, helm descriptor and it doesn't either create the Git repo.

### Steps to reproduce the behavior

I copy/paste the whole command

	> jx create spring -d web -d actuator --verbose=true
	? Language: java
	? Group: com.group
	? Artifact: jx-demo-spring-boot
	Created spring boot project at C:\Users\nicolas-delsaux\Documents\Group\jx-demo-spring-boot
	error: secrets "jenkins" not found

### Jx version

The output of `jx version` is:

```
NAME               VERSION
jx                 1.3.185
jenkins x platform 0.0.2266
kubernetes cluster v1.9.7-gke.6
kubectl            v1.10.3
helm client        v2.9.1+g20adb27
helm server        v2.10.0+g9ad53aa
git                git version 2.17.0.windows.1
```

### Kubernetes cluster

What kind of Kubernetes cluster are you using & how did you create it?

It's a Google Kubernetes Engine cluster deployed through an Ansible script

### Operating system / Environment

Windows 10

### Expected behavior

The aforementionned files should be created as well as the git repo 

### Actual behavior

Only the java source files and maven pom are created.

**Comment by rawlingsj** [comment]
Did you create the cluster or was it created by someone else?
Are you in your team namespace?  By default this is `jx` you can run `jx ns jx` to switch.

**Comment by Riduidel** [comment]
The cluster was created using jx create cluster gcp on another machine.

I was not aware there were team namespaces, as a consequence I can confidently say "no, I was not in my team namespace". In fact, the "developping" section on the doc has no points regarding any namespaces. As a consequence I supposed I could create my spring boot application as soon as jx was able to speak to my K8s cluster. Is there any additionnal steps ?

**Comment by rawlingsj** [comment]
ok yeah if you run `jx ns jx` that will change you into the team namespace.  You should then be able to run commands like `jx create quickstart` `jx import` etc.

**Comment by Riduidel** [comment]
Seems like installing Jenkins in my environment from a container which hadn't chrome installed was a *bad* idea.

When using install scripts on my Windows machine through git bash (since some components - helm, istio - seems to rely upon bash conventions), everything works OK. You can close issue if you think there is no work required to improve state detection on developer machine.

**Comment by meghalidhoble** [comment]
Hi @Riduidel , I am seeing similar issue, even though I have all the setup for kubernetes and helm up and running. Even the pods seen up. When I run the "jx start pipeline" command it gives me this error. 
I do have my jenkins server running at 8080 port. 

~# jx start pipeline -t
error: secrets "jenkins" not found

~# service jenkins status
â jenkins.service - LSB: Start Jenkins at boot time
   Loaded: loaded (/etc/init.d/jenkins; bad; vendor preset: enabled)
   Active: active (exited) since Fri 2018-11-23 12:16:30 IST; 3h 52min ago
     Docs: man:systemd-sysv-generator(8)
  Process: 6663 ExecStop=/etc/init.d/jenkins stop (code=exited, status=0/SUCCESS)
  Process: 6714 ExecStart=/etc/init.d/jenkins start (code=exited, status=0/SUCCESS)
    Tasks: 0
   Memory: 0B
      CPU: 0

Any suggestions, what is missing for me?

**Comment by ccojocar** [comment]
@meghalidhoble The jenkins installation seems to be incomplete. Do you have the `jenkins` secret?

`kubectl get secret -n jx jenkins`?

**Comment by ccojocar** [comment]
@Riduidel I am going to close this issue since the requested evidence weren't provided, and it seems to be outdated. Feel free to open another issue if you are still facing the problem.

**Comment by lukaszlenart** [comment]
Hi,
I have similar problems with using `jx import`, cluster was created by our devops who left the project. Each time I'm trying to import a new project into Jenkins I get:

```
error: updating credentials "https://github.com" in Jenkins: creating credentials "https://github.com" in Jenkins: 401 Unauthorized
```

here is `jx` version I'm running

```
NAME               VERSION
jx                 2.0.1240
Kubernetes cluster v1.14.9-eks-502bfb
kubectl            v1.16.3
helm client        2.16.3
git                2.25.0
Operating System   Mac OS X 10.15.3 build 19D76
```

I have run `kubectl get secret -n jx jenkins` but that didn't help, I got a copy of `~/.jx/jenkinsAuth.yaml` but that didn't work as well. Even if I removed `~/.jx` and started from scratch only `gitAuth.yaml` is created, there is no `jenkinsAuth.yaml`.

Is there a way to take control over the cluster? Basically we are not able to manage the cluster anymore :(
